.ci/scripts/distribution/random-test-java.sh

```diff
@@ -10,19 +10,20 @@ JUNIT_THREAD_COUNT=${JUNIT_THREAD_COUNT:-}
 MAVEN_PROPERTIES=(
   -DskipITs
   -DskipChecks
-  -DtestMavenId=1
+  -DtestMavenId=4
```

Comment:
:question:  I'm not sure where this is coming from & what it means. Can you explain? :)

Reply:
It's used in our `io.camunda.zeebe.test.util.socket.SocketUtil` class. It's used to allocate specific port ranges to each stage/fork to ensure there are no port collisions for tests running in the same pod.

Reply:
Ah, and now we have to increase it because regular tests are running at the same time?